sdk/src/wallet/operations/output_consolidation.rs

```
@@ -116,7 +116,6 @@ where
                     .map(|bech32| bech32.into_inner())
                     .unwrap_or_else(|| wallet_address.into_inner()),
             ),
-            allow_additional_input_selection: false,
```

I dunno about this...I know the test was failing but this was put here on purpose. I feel like we should not consolidate more outputs than we requested

---

We provided the inputs in Stardust as no other inputs were required, but now we need Mana and these outputs will often not have enough Mana for the consolidation transaction
We could just return an error, but I think it makes more sense to transition a chain output to provide the Mana if possible

---

alright 🤷‍♀️